### PR TITLE
I18n

### PR DESCRIPTION
**Pull request by mmis1000**
Enable I18n globally and make languageService instance private  

Add dependency of 
 - sponge-i18n 0.1.3
   - packet gate 0.1.2

Checklist
 - [x] items
   - [x] Weapon
   - [x] Materials
   - [x] Tools
   - [x] Upgrades
   - [x] Fakeblock item
 - [x] gui
   - [x] MainMenu and all child gui
   - [x] PlanetTerminal and all child gui
   - [x] chunk loader
   - [x] Hi-tech crafting recipe and all child gui
 - [x] Messages
   - [x] create planet
   - [x] chat confirm
   - [x] sign edit
   - [x] member remove notice
 - [x] Misc
   - [x] armor status message
   - [x] star dust event

### CODE REVIEW

File: src/main/kotlin/one/oktw/galaxy/gui/GalaxyJoinRequest.kt [hunk 1]
```diff
@@ -29,11 +28,11 @@ import kotlin.streams.toList
 
 class GalaxyJoinRequest(private val galaxy: Galaxy) : PageGUI<UUID>() {
     private val userStorage = Sponge.getServiceManager().provide(UserStorageService::class.java).get()
-    private val lang = languageService.getDefaultLanguage()
+    private val lang = Main.translationService
     override val token = "InviteManagement-${galaxy.uuid}"
-    override val inventory: Inventory = Inventory.builder()
+    override val inventory: Inventory= Inventory.builder()
```
Comment (james58899) [inline src/main/kotlin/one/oktw/galaxy/gui/GalaxyJoinRequest.kt]: missing space

File: src/main/kotlin/one/oktw/galaxy/gui/UpgradeSlot.kt [hunk 2]
```diff
@@ -10,15 +10,19 @@ import org.spongepowered.api.item.inventory.Inventory
 import org.spongepowered.api.item.inventory.InventoryArchetypes
 import org.spongepowered.api.item.inventory.Slot
 import org.spongepowered.api.item.inventory.property.InventoryTitle
-import org.spongepowered.api.text.Text
 import java.util.*
 
-class UpgradeSlot(parent: GUI, private var upgrade: List<Upgrade>, private vararg val acceptType: UpgradeType) : GUI() {
+class UpgradeSlot(
```
Comment (james58899) [inline src/main/kotlin/one/oktw/galaxy/gui/UpgradeSlot.kt]: dont change this line

File: src/main/kotlin/one/oktw/galaxy/gui/CreatePlanet.kt [hunk 3]
```diff
@@ -84,49 +82,49 @@ class CreatePlanet(private val galaxy: Galaxy) : GUI() {
...
+        val name = input(player, Text.of(AQUA, lang.of("Respond.createPlanetInputName")).toLegacyText(player))?.toPlain()
 
         if (name == null) {
-            player.sendMessage(Text.of(RED, "已取消創建星球"))
+            player.sendMessage(Text.of(RED, lang.of("Respond.createPlanetCancel")).toLegacyText(player))
             return false
         }
 
-        if (confirm(player, Text.of(AQUA, "確定要創建名為「", TextColors.RESET, BOLD, name, TextStyles.RESET, AQUA, "」的星球嗎？")) == true) {
+        if (confirm(player, Text.of(AQUA,  lang.of("Respond.createPlanetConfirmName", BOLD, name)).toLegacyText(player)) == true) {
```
Comment (jimchen5209) [inline src/main/kotlin/one/oktw/galaxy/gui/CreatePlanet.kt]: 星球名稱變成Bold了

他現在看起來長這樣
default

Comment (mmis1000) [reply src/main/kotlin/one/oktw/galaxy/gui/CreatePlanet.kt]: my bad，寫錯東西

File: src/main/kotlin/one/oktw/galaxy/Main.kt [hunk 4]
```diff
@@ -82,12 +84,20 @@ class Main {
...
     fun onPreInit(event: GamePreInitializationEvent) {
         serverThread = Sponge.getScheduler().createSyncExecutor(this).asCoroutineDispatcher()
         delay = DelayedExecute(plugin)
-        languageService = LanguageService()
-        vanillaLanguageService = LanguageService("vanilla")
+
+        val languageService = LanguageService()
+        val vanillaLanguageService = LanguageService("vanilla")
+
+        Sponge.getServiceManager().provide(one.oktw.i18n.api.I18n::class.java).get().let {
```
Comment (james58899) [inline src/main/kotlin/one/oktw/galaxy/Main.kt]: use .also

File: src/main/kotlin/one/oktw/galaxy/Main.kt [hunk 5]
```diff
@@ -82,12 +84,20 @@ class Main {
         event.register(NetherGenModifier())
     }
 
+    @Suppress("DEPRECATION")
```
Comment (james58899) [inline src/main/kotlin/one/oktw/galaxy/Main.kt]: remove

File: src/main/kotlin/one/oktw/galaxy/Main.kt (hunk 4 above)
Comment (mmis1000) [reply src/main/kotlin/one/oktw/galaxy/Main.kt]: 為啥？這裡就依個參數啊？設this做什麼？

Comment (james58899) [reply src/main/kotlin/one/oktw/galaxy/Main.kt]: also是沒有回傳值的，一樣有it

Comment (mmis1000) [reply src/main/kotlin/one/oktw/galaxy/Main.kt]: 還是你是想說apply？

**Pull request merged**